src/main/java/net/sf/jabref/importer/CustomImporter.java

```diff
@@ -117,6 +117,10 @@ public URL getBasePathUrl() throws MalformedURLException {
 
     @Override
     public boolean equals(Object o) {
+        if (o == null) {
+            return false;
+        }
+
         boolean equalName = this.getName().equals(((CustomImporter) o).getName());
```

Comment:
The equals methods looks a bit weird, it should follow a more clear pattern. A possible good solution would be this:
```
        @Override
        public boolean equals(Object other)
        {
            if (other == null) // Null-check
                return false;

            if (this == other) // reflexivity 
                return true;

            if (this.getClass() != other.getClass()) // Type safety 
                return false;

            final CustomImpoter importer = (CustomImporter) other;
            return Objects.equals() && ...
        }
````